Auto-Install-win.bat

```diff
@@ -0,0 +1,77 @@
+@echo off
+
+REM Set paths
+set EXE_PATH=%CD%\jellyfin-rpc\jellyfin-rpc.exe
+set JSON_PATH=%CD%\jellyfin-rpc\main.json
+set DOWNLOAD_URL=https://github.com/Radiicall/jellyfin-rpc/releases/latest/download/jellyfin-rpc.exe
+set DOWNLOAD_DIR=jellyfin-rpc
```

Comment:
The script **always** checks %APPDATA%\jellyfin-rpc for the main.json file, this is set in code if you look at the README. This line has to be changed to %APPDATA%\jellyfin-rpc.

Reply:
Miscommunication, the application always checks appdata for the config file, if you want it somewhere else then you have to use `-c` and seeing as this is going into the main branch then i think it should use the default locations..